src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -44,9 +43,9 @@ public AddCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
-        Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+        Tier tier = ParserUtil.parseTag(argMultimap.getValue(PREFIX_TAG).orElse("NA"));
```

Comment:
I like this default value. We just have to remember to document this down and possibly modify the UI to not show the label, if this is the text value for tier.